### PR TITLE
added international elements form page

### PR DESCRIPTION
**Pull request by kurt-uk**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/secure/RapidBoard.jspa?rapidView=789&projectKey=FPL&view=detail&selectedIssue=FPL-11

### Change description ###
added international elements form page, created tests and updated the readme

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/paths/enterInternationalElements_test.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	caseViewPage.goToNewActions(config.applicationActions.enterInternationalElement);
+	I.waitForElement('ccd-case-edit-page', 10);
+});
+
+Scenario('test form save with no details entered', (I) => {
+	I.continueAndSubmit(config.eventSummary, config.eventDescription);
+	I.see(`updated with event: ${config.applicationActions.enterInternationalElement}`);
+});
+//
+// Scenario('test half form filled', (I, enterInternationalElementsPage) => {
```
Comment (geek-soft) [inline e2e/paths/enterInternationalElements_test.js]: Please remove commented out code.

Comment (geek-soft) [reply e2e/paths/enterInternationalElements_test.js]: Actually we need test that tests data can be entered and then is visible in one of the view tabs.

File: README.md [hunk 2]
```diff
@@ -25,6 +25,11 @@ Add users and roles:
 sh addUsersAndRoles.sh
 ```
 
+Add the spreadsheet:
+```
+./bin/ccd-import-definition.sh ~/fpl-ccd-configuration/FPL_CCD_DEFINITION_TEMPLATEx.xlsx
```
Comment (geek-soft) [inline README.md]: A `./bin/ccd-import-definition.sh` does not exist in project directory. Please correct path. Also `~/fpl-ccd-configuration/` part won't work for everyone.

**Comment by geek-soft** [comment]
@kurt-uk can you also add story ID to PR message. Also make sure story ID ends in merge commit message. Having story ID in repo will help tracing code changes.

File: README.md (hunk 2 above)
Comment (swalker125) [reply README.md]: ./ccd-docker/bin ...

File: e2e/pages/internationalElements/enterInternationalElements.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+const I = actor();
+
+module.exports = {
+
+	fields: {
+
```
Comment (swalker125) [inline e2e/pages/internationalElements/enterInternationalElements.js]: blank line

File: e2e/paths/enterInternationalElements_test.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Feature('Enter international elements');
+
+Before((I, caseViewPage) => {
+	I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventDescription);
+	I.waitForElement('.tabs', 10);
+	caseViewPage.goToNewActions(config.applicationActions.enterInternationalElement);
+	I.waitForElement('ccd-case-edit-page', 10);
+});
+
+Scenario('test form save with no details entered', (I) => {
```
Comment (swalker125) [inline e2e/paths/enterInternationalElements_test.js]: do you need the word test at the beginning of your scenarios?

Comment (swalker125) [reply e2e/paths/enterInternationalElements_test.js]: would also reword this slightly. You aren't testing the form is saved you're testing the c110a application is updated

File: e2e/paths/enterInternationalElements_test.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+	I.continueAndSubmit(config.eventSummary, config.eventDescription);
+	I.see(`updated with event: ${config.applicationActions.enterInternationalElement}`);
+});
+
+Scenario('test half form filled', (I, enterInternationalElementsPage) => {
```
Comment (swalker125) [inline e2e/paths/enterInternationalElements_test.js]: similar to above

File: e2e/paths/enterInternationalElements_test.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+	enterInternationalElementsPage.halfFillForm();
+	I.see('Give reason');
+	I.continueAndSubmit(config.eventSummary, config.eventDescription);
+	I.see(`updated with event: ${config.applicationActions.enterInternationalElement}`);
+});
+
+Scenario('test form is fully filled in', (I, enterInternationalElementsPage) => {
```
Comment (swalker125) [inline e2e/paths/enterInternationalElements_test.js]: similar to above

File: e2e/paths/selectHearing_test.js [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
 
 Before((I, caseViewPage) => {
 	I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventDescription);
 	I.waitForElement('.tabs', 10);
 	caseViewPage.goToNewActions(config.applicationActions.selectHearing);
-	I.waitForElement('.button', 10);
+	I.waitForElement('ccd-case-edit-page', 10);
 });
 
 Scenario('test form save with no details entered', (I) => {
```
Comment (swalker125) [inline e2e/paths/selectHearing_test.js]: I know this has already got through, but same issues as international elements

**Pull request merged**